packages/@dcl/sdk/tsconfig.cli.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Is it really needed to have two tsconfigs? everything in the src folder is compiled